src/core/templatetags/due_dates.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from django.utils import timezone
+
+
+register = template.Library()
+
+@register.simple_tag
+def due_date(days=0, input_type="date"):
+    """
+    Get a string representing today's date or the now datetime,
+    with an offset of X days.
```

Comment:
Adding a usage section in here to show folks how to use the tag would also be good.